searchcore/src/vespa/searchcore/proton/docsummary/docsumcontext.cpp

```diff
@@ -34,6 +35,10 @@ namespace {
 Memory DOCSUMS("docsums");
 Memory DOCSUM("docsum");
 
+void addTimedOut(Inserter & inserter, fastos::TimeStamp left) {
+    inserter.insertObject().setString("error", vespalib::make_string("Timeout at %ldus", left.us()));
```

Comment:
The error message seems a bit confusing. ('Timeout at -42')

Reply:
Also consider the effect of injecting a synthetic 'error' field. Will it conflict with document fields or simply be discarded in the qrs because it is not in the list of wanted fields etc.

Reply:
Yes, we should discuss this at architect meeting today.